lib/tasks/attempts.rake

```diff
@@ -78,6 +81,12 @@ namespace :attempts do
     puts 'Remember to restart Rails after updating application.yml.default!' if failed
   end
 
+  desc 'Enable irs_attempts_api_enabled for Sinatra SP'
+  task enable_for_sinatra: :environment do
+    sp = ServiceProvider.find_by(friendly_name: 'Example Sinatra App')
+    sp.update(irs_attempts_api_enabled: true)
+  end
```

Comment:
Since we're already checking this and telling you if it's wrong, let's just fix it directly rather than making the user go off to the rails console.

I briefly debated adding a check so this would refuse to run in prod, but (1) you shouldn't run random rake tasks in prod, (2) there probably isn't an actual SP named "Example Sinatra App" in prod, and (3) if there is, someone running this probably actually intends to enable it? 🤷 Happy to add a check if people believe it's warranted.

Reply:
I'd consider maybe making the friendly name (or even issuer?) an argument so we ca call `rake enable_attempts_api ISSUER=ABCDEF`

Reply:
That'd be easy enough to implement, but the intent here was just for localhost use with the default Sinatra app. I think if people are going to go edit other providers they're probably better off using the Rails console.